NitroxClient/MonoBehaviours/Gui/MainMenu/MainMenuMultiplayerPanel.cs

```diff
@@ -160,17 +160,24 @@ private void LoadSavedServers()
                     string serverName = lineData[0];
```

Comment:
Let's use the json serializer.

Reply:
I agree, this is nonsense. We will be playing patty cake with this stupid flat file forever if we keep this up.